tests/unit_tests/test_data_photon.py

```diff
@@ -144,3 +145,9 @@ def test_export_to_hdf5(tmpdir, element):
            element2.bremsstrahlung['electron_energy']).all()
     # Export to hdf5 again
     element2.export_to_hdf5(filename, 'w')
+
+def test_photodat_only(run_in_tmpdir):
+    endf_dir = Path(os.environ['OPENMC_ENDF_DATA'])
+    photoatomic_file = endf_dir / 'photoat/photoat-001_H_000.endf'
+    data=openmc.data.IncidentPhoton.from_endf(photoatomic_file)
```

Comment:
For Python code, we follow PEP8 guidelines:
```suggestion
    data = openmc.data.IncidentPhoton.from_endf(photoatomic_file)
```